README.md

```diff
@@ -44,11 +44,18 @@ To bring up a dev environment, run `make docker-dev`.
 This brings up an additional vault service, used for integration testing of
 some auxiliary binaries.
 
-You can run the unit and integration tests via `make docker-test`
+### Testing
 
-Once you are in the Docker container you can run the web server via `go run bin/grant-server/main.go`.
+## Automated
+You can run the unit and integration tests via `make docker-test`. (Accurate?)
```

Comment:
Wasn't sure if `Automated` is the right terminology here. Maybe `Default Testing Behavior`?

Reply:
Default Testing Behavior sounds good.